tests/unit/tasks/apiRequests/get.ts

```diff
@@ -0,0 +1,32 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import nock from 'nock';
+import { createClient, rqlBuilder } from '../../../../src';
+import { TASKS_BASE } from '../../../../src/constants';
+import { apiRequests } from '../../../__helpers__/apiRequests';
+
+describe('Tasks - API Requests', () => {
+  const host = 'https://api.xxx.extrahorizon.com';
+  const exh = createClient({
+    host,
+    clientId: '',
+  });
+
+  it('Lists API Requests matching an RQL query', async () => {
```

Comment:
Previously I had tested the pagination methods incredibly deeply for the loginAttempts, we agreed it would be unnecessary to do it again as they're just copy paste into the interfaces.

Thus I have just tested the construction / targeting of the correct urls here.